pkg/client/proxy.go

```diff
@@ -17,9 +18,18 @@ import (
 )
 
 var (
-	ErrParameter = errors.Errorf("missing required parameter")
+	ErrParameterFmt = "missing required %v parameter"
 )
 
+func checkParameters(input map[string]string) error {
```

Comment:
Can we rename to `validateProxyMethodParameters`?

Reply:
Updated.